### PR TITLE
Gallery example improvement

### PR DESCRIPTION
**Pull request by rougier**
The very first example in the gallery is barely visible. I tried to reverse colors and add some titles to make it more explicit what the images are representing.

Here is a screenshot:

figure_1

**Note**: I did not find the "official" way of reversing color in an aimage so I used 1-image. This works only if image channels are normalized.

### CODE REVIEW

**Comment by stefanv** [comment]
I'm surprised to see this; I thought we had removed all examples containing lena.  Typically, we switch to `data.astronaut()`.

**Comment by jni** [comment]
@rougier We actually tried to scrub every mention of `lena` from our docs! Given your improvement's nature, no wonder we missed this one! Could you replace the image with `data.astronaut()`?

**Comment by jni** [comment]
@stefanv @rougier LOL

**Comment by vighneshbirodkar** [comment]
@rougier  Could you instead normalize the out for better visibility ?

**Comment by rougier** [comment]
@vighneshbirodkar You mena keeping the white over black and normalize the output ?
Is there a skimage function for normalization (or should I do it by hand) ?

**Comment by vighneshbirodkar** [comment]
@rougier You should do that by hand. And yes, I mean keeping white over black.

**Comment by jni** [comment]
@rougier @vighneshbirodkar not by hand. `skimage.exposure.rescale_intensity`

**Comment by vighneshbirodkar** [comment]
@jni @stefanv 
```
vighnesh@viggie-pc:scikit-image$ git grep " lena "
skimage/data/tests/test_data.py:    lena = data.lena()
skimage/feature/tests/test_match.py:    lena image and its rotated version with the expected keypoint pairs with
skimage/feature/tests/test_match.py:    lena image and its rotated version with the expected keypoint pairs with
skimage/morphology/tests/test_grey.py:bw_lena = lena > 100
```

**Comment by jni** [comment]
@vighneshbirodkar those are hard to get rid of because they are tests hardcoded to match values from Matlab. But, as usual... PRs welcome! =P

**Comment by vighneshbirodkar** [comment]
@jni Even `test_grey.py` is hard to fix ?

**Comment by rougier** [comment]
Ok, I'm lost now...

**Comment by jni** [comment]
@rougier ignore us! Just use `skimage.exposure.rescale_intensity` to ensure maximum visibility out of the filters!

**Comment by rougier** [comment]
Output:

figure_1

**Comment by vighneshbirodkar** [comment]
@rougier Did you try leaving the background black with normalizing ?

**Comment by rougier** [comment]
Nope, here is the alternate version:
figure_1

I'll let you vote who which is the best (I prefer the white version personally)

**Comment by jni** [comment]
@rougier I agree, white is better.

**Comment by vighneshbirodkar** [comment]
@rougier @jni 
Are we OK with the inverting colors, for example, from the background to her suit, there is a positive orange color difference, but in the white case, the output leads us to believe her suit is blue .

I hope that made sense

**Comment by jni** [comment]
@vighneshbirodkar not clear at all, sorry! =\ Either way I think the answer is yes. These colorized edge images don't make much visual sense to me, either way.

**Comment by rougier** [comment]
Same for me.  I do not really know what is the expected theoretical result.

**Comment by vighneshbirodkar** [comment]
Hmmm, I guess that's right, there is no expected result here.

**Comment by emmanuelle** [comment]
+1 for white version

**Comment by JDWarner** [comment]
I'll grant the white version conveys color more effectively. However, for the sake of discussion - and as someone involved in medical imaging - I prefer the black version. It's more intuitive.

Black in a normal image means minimum value (often but not always zero). To me, the white variant looks inverted, and I fear it might come across as unintuitive. It implies the filter results are maximized in flat areas, with smaller values (darker) at edges.

**Comment by rougier** [comment]
Black or white in the end ? @JDwarner comment makes sense to me (even if I still prefere the white version, but it is purely aesthetic)

**Comment by emmanuelle** [comment]
I don't have strong opinions on black vs. white, both solutions will be an improvement compared to how it looks now. However, the white version still looks much better to me.

File: doc/examples/plot_adapt_rgb.py [hunk 1]
```diff
@@ -42,13 +42,24 @@ def sobel_hsv(image):
...
+from skimage.exposure import rescale_intensity
 import matplotlib.pyplot as plt
 
-image = data.lena()
+image = data.astronaut()
 
-fig, (ax_each, ax_hsv) = plt.subplots(ncols=2)
-ax_each.imshow(sobel_each(image))
-ax_hsv.imshow(sobel_hsv(image))
+fig = plt.figure( figsize=(14,7) )
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: nitpicking, sorry :-), but there are some pep8 issues
```fig = plt.figure(figsize=(14, 7))```

File: doc/examples/plot_adapt_rgb.py [hunk 2]
```diff
@@ -42,13 +42,24 @@ def sobel_hsv(image):
...
+ax_each = fig.add_subplot(121)
+ax_hsv  = fig.add_subplot(122)
+
+# We use 1 - sobel_each(image) but this will not work if image is not normalized
+ax_each.imshow( rescale_intensity(1-sobel_each(image)))
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: pep8: ```ax_each.imshow(rescale_intensity(1 - sobel_each(image)))```

File: doc/examples/plot_adapt_rgb.py [hunk 3]
```diff
@@ -42,13 +42,24 @@ def sobel_hsv(image):
...
+ax_each.set_xticks([]), ax_each.set_yticks([])
+ax_each.set_title("Sobel filter computed\n on individual RGB channels")
+
+# We use 1 - sobel_hsv(image) but this will not work if image is not normalized
+ax_hsv.imshow( rescale_intensity((1-sobel_hsv(image))))
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: ```ax_hsv.imshow(rescale_intensity(1 - sobel_hsv(image)))``` (pep8 + I think that a couple of parentheses were not needed

File: doc/examples/plot_adapt_rgb.py [hunk 4]
```diff
@@ -94,8 +105,13 @@ def as_gray(image_filter, image, *args, **kwargs):
 def sobel_gray(image):
     return filters.sobel(image)
 
-fig, ax = plt.subplots()
-ax.imshow(sobel_gray(image), cmap=plt.cm.gray)
+fig = plt.figure( figsize=(7,7) )
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: pep8

File: doc/examples/plot_adapt_rgb.py [hunk 5]
```diff
@@ -94,8 +105,13 @@ def as_gray(image_filter, image, *args, **kwargs):
...
+ax = fig.add_subplot(111)
+
+# We use 1 - sobel_gray(image) but this will not work if image is not normalized 
+ax.imshow( rescale_intensity(1-sobel_gray(image)), cmap=plt.cm.gray)
```
Comment (emmanuelle) [inline doc/examples/plot_adapt_rgb.py]: pep8

**Comment by emmanuelle** [comment]
Thank you Nicolas, I added a couple of minor remarks about pep8. We're getting there :-)

**Comment by emmanuelle** [comment]
Merged, thank you very much @rougier !

**Pull request merged**